src/Keboola/DbWriter/Snowflake/Application.php

```diff
@@ -30,7 +31,11 @@ public function runAction()
             if ($table['incremental']) {
                 $table['dbName'] = $writer->generateTmpName($table['dbName']);
             }
-            $table['items'] = $this->reorderColumns($manifest['columns'], $table['items']);
+
+            $table['items'] = $this->reorderColumns(
```

Comment:
tohle je definovanej v common?

Reply:
Snowflake writer ma staging v S3, proto to ma tady vlastni implementaci.

```
        "staging_storage": {
          "input": "s3"
        },
```